CMakeLists.txt

```diff
@@ -34,8 +34,6 @@ endmacro ()
 
 set(CMAKE_EXPORT_COMPILE_COMMANDS 1) # Write compile_commands.json
 set(CMAKE_LINK_DEPENDS_NO_SHARED 1) # Do not relink all depended targets on .so
-set(CMAKE_CONFIGURATION_TYPES "RelWithDebInfo;Debug;Release;MinSizeRel" CACHE STRING "" FORCE)
-set(CMAKE_DEBUG_POSTFIX "d" CACHE STRING "Generate debug library name with a postfix.")    # To be consistent with CMakeLists from contrib libs.
```

Comment:
it looks we need set `CMAKE_DEBUG_POSTFIX` to empty

```
set(CMAKE_DEBUG_POSTFIX "" CACHE STRING "Generate debug library name with a postfix.") # To be consistent with CMakeLists
````

Otherwise, building delta-kernal-rs will end with  `error: could not compile openssl-sys (lib) due to 1 previous error warning: build failed, waiting for other jobs to finish..`

Reply:
Please share your full cmake command and error messages. I can't reproduce it (and neither does the CI)

Reply:
@Algunenano ,  It's just **Debug** Build from clean cmake build folder.

Reply:
```
/mnt/ch/ClickHouse $ mkdir build_debug
/mnt/ch/ClickHouse $ cd build_debug/
/mnt/ch/ClickHouse/build_debug $ cmake .. -DCMAKE_BUILD_TYPE=Debug
...
/mnt/ch/ClickHouse/build_debug $ ninja ./contrib/delta-kernel-rs-cmake/libdelta_kernel_ffi.a
[0/2] Re-checking globbed directories...
[1251/1254] cd /mnt/ch/ClickHouse/contrib/delta-kernel-rs/ffi && /usr/bin/cmake -E env "CFLA...ebug/contrib/corrosion-cmake/config.toml --offline --locked -- -Cdefault-linker-libraries=no
   Compiling proc-macro2 v1.0.93
...
   Compiling parquet v54.2.0
    Finished `dev` profile [unoptimized + debuginfo] target(s) in 22.60s
[1253/1254] Running utility command for _cargo-build_delta_kernel_ffi
```

It works. So please, share your full cmake command and full output

Reply:
@Algunenano thanks for you share. Now, I have found that simply rebuilding is not working, and I need to regenerate the CMake